specs/trip_dispatch_spec.rb

```diff
@@ -88,5 +88,53 @@
       passenger.must_be_instance_of RideShare::Passenger
       passenger.trips.must_include trip
     end
-  end
-end
+
+  end # end of describe "loader methods"
+
+  describe "find_available_driver method" do
+
+    it "finds the first driver who has an available status" do
```

Comment:
I like these tests! One observation: in a real-world application, these helper methods would probably be marked `private`, and testing this functionality would go through the public method, `request_trip`.